pkg/client/unversioned/services.go

```diff
@@ -98,3 +99,16 @@ func (c *services) Watch(label labels.Selector, field fields.Selector, resourceV
 		FieldsSelectorParam(field).
 		Watch()
 }
+
+func (c *services) ProxyGet(name, path string, params map[string]string) ([]byte, error) {
```

Comment:
Calling Raw() is somewhat risky for proxy calls - it means the client is going to hold everything in memory and we have no boundaries on the size of what Raw() can return today.